release/air_tests/air_benchmarks/mlperf-train/file_size_benchmark.sh

```diff
@@ -1,42 +1,63 @@
 #!/bin/bash
-# Test Ray Data bulk ingestion performance as the size of input files change.
+# Test Ray Data vs. tf.data bulk ingestion performance as the size of input
+# files changes.
 
 # Exit if any of the test commands fail.
 set -x -e pipeline
 
-NUM_IMAGES_PER_FILE=${NUM_IMAGES_PER_FILE:-"32 512 8192"}
-MIN_PARALLELISM=10
-NUM_EPOCHS=1
-BATCH_SIZE=64
+BATCH_SIZE=32
 SHUFFLE_BUFFER_SIZE=0
 DATA_DIR=/home/ray/data
-
-MAX_IMAGES_PER_FILE=$( \
-    echo "$NUM_IMAGES_PER_FILE" \
-        | sed 's/ /\n/g' \
-        | python -c "import sys; print(max([int(line) for line in sys.stdin]))")
-NUM_IMAGES_PER_EPOCH=$((MIN_PARALLELISM * MAX_IMAGES_PER_FILE))
-
 SHARD_URL_PREFIX=https://air-example-data.s3.us-west-2.amazonaws.com/air-benchmarks
 
-for num_images_per_file in $NUM_IMAGES_PER_FILE; do
-    num_files=$(python -c "import math; print(math.ceil($NUM_IMAGES_PER_EPOCH/$num_images_per_file))")
-    
+# First epoch is for warmup, report results from second.
```

Comment:
i'm unclear on why we run 2 epochs and report results from the second one. what does it mean for the first epoch to be a "warmup"?

Reply:
Ah the system might still be warming up on the first epoch, so that's why we're only reporting results for the second.

Reply:
(like starting Ray workers, etc)